textalyze.js

```diff
@@ -13,9 +13,21 @@
  */
 function itemCounts(array) {
   const counts = new Map();
+  var element = "";
+  var count = 0;
 
   // Your code here.
   // Run 'npm test' to see what tests need to pass.
+  for(var i = 0; i < array.length; i++) {
```

Comment:
Right now you're looping over `array` once for every element in `array`.  Can you loop over it only once?

Reply:
I'm gonna try that as soon as I finish the iteration I'm on at the moment.